cartographer/transform/rigid_transform_test_helpers.h

```diff
@@ -40,8 +40,8 @@ Eigen::Transform<T, 3, Eigen::Affine> ToEigen(const Rigid3<T>& rigid3) {
 }
 
 MATCHER_P2(IsNearly, rigid, epsilon,
-           std::string(std::string(negation ? "isn't" : "is", " nearly ") +
-                       rigid.DebugString())) {
+           std::string(std::string(negation ? "isn't" : "is") +
```

Comment:
Some of the "std::string()" looks redundant:

Maybe: std::string(negation ? "isn't nearly" : "is nearly")  + std::string(rigid.DebugString());
Or even: std::string(negation ? "isn't nearly" : "is nearly")  + rigid.DebugString();

Reply:
Done (the latter)